### ISSUE TITLE
Energy issue in `MuReNNDirect`

### ISSUE THREAD

**Issue by xir4n** [kymatio/murenn#52]
1. to make the output signal's energy approximately at the same level crossing scales
2. a huge amount of negative valued signal appeared after `down_j`

see code below
```python
J = 8
Q = 1
T = 10
murenn_layer = MuReNNDirect(
    J=J, 
    Q=Q, 
    T=T, 
    in_channels=1,
)
# shape (B, C, Q, J, T)
Y = murenn_layer(X).detach()

# permute to (C, Q, J, T, B)
Y = Y.permute((1, 2, 3, 4, 0))

# reshape to (C*Q*J, T*B)
Y  = Y.reshape((Q*J, -1))
plt.boxplot(Y)
``` 
result
![image](https://github.com/kymatio/murenn/assets/97965893/a6d079bc-2e9c-4156-9eb7-627d6c513706)

**Comment by lostanlen** [comment]
This is a serious bug. I recommend writing a unit test for MuReNNDIrect` to make sure this doesn't happen

**Comment by lostanlen** [comment]
closed by #55
